docs: mention empty string in `userDataDir` (#9069)
microsoft/playwright: docs/src/api/class-browsertype.md

Path to a User Data Directory, which stores browser session data like cookies and local storage. More details for
[Chromium](https://chromium.googlesource.com/chromium/src/+/master/docs/user_data_dir.md#introduction) and
[Firefox](https://developer.mozilla.org/en-US/docs/Mozilla/Command_Line_Options#User_Profile).
Note that Chromium's user data directory is the **parent** directory of the "Profile Path" seen at `chrome://version`. Use an empty string to

> generate and destroy it automatically.

Author: ```suggestion
set a temporary directory instead.
```

> Note that Chromium's user data directory is the **parent** directory of the "Profile Path" seen at `chrome://version`. Use an empty string to
> generate and destroy it automatically.

Author: ```suggestion
Note that Chromium's user data directory is the **parent** directory of the "Profile Path" seen at `chrome://version`. Pass an empty string to
use a temporary directory instead.
```
